items/materials/avalipaneling.matitem

```diff
@@ -6,5 +6,7 @@
   "description" : "Light and airy panels.",
   "shortdescription" : "Horizontal Panels",
 
-  "materialId" : 12508
+  "materialId" : 12508,
+  
+  "learnBlueprintsOnPickup" : [ "avalipaneling" ]
```

Comment:
Unnecessary: Lathe and Loom don't require recipes, so you don't need to teach the recipes to the player